x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -931,6 +932,149 @@ This will output:
 ]
 ----
 
+[[chain]]
+[float]
+=== `chain`
```

Comment:
The docs contain a sequence diagram that explains the high level flow. I think the chain feature should be added to the diagram.